src/react/BlockSourcePage.jsx

```diff
@@ -90,7 +90,7 @@ export default ({ astarte, history, blockId }) => {
             </Row>
           </ContentCard>
           <Row className="justify-content-end m-3">
-            {!block.isNative && (
+            {!block.beamModule && (
```

Comment:
checking for a field to infer the type of a variable is a bad practice. Let's use instanceOf or add a field `type` to our models